packages/create-react-app/createReactApp.js

```diff
@@ -268,6 +274,11 @@ function createApp(name, verbose, version, template, useYarn, usePnp) {
     version: '0.1.0',
     private: true,
   };
+
+  if (yarnInfo.hasMaxYarnPnp) {
+    packageJson.packageManager = `yarn@${yarnInfo.yarnVersion}`;
```

Comment:
I was thinking the `packageManager` field should be set regardless of package manager and/or version.

Reply:
I want to make this change focused on improving Yarn 2+ support, without affecting Yarn 1.x and other package managers at all